src/views/News/index.js

```diff
@@ -19,530 +23,764 @@ import {
-        content = (
-          <NewsList
-            news={this.state.filteredNews}
-            personalUnapprovedNews={this.state.personalUnapprovedNews}
-            updateSnackbar={this.updateSnackbar}
-            currentUsername={this.state.currentUsername}
-            callFunction={this.callFunction}
-          />
-        );
-      } else {
-        content = (
-          <Typography variant="h4" align="center">
-            No News To Show
-          </Typography>
-        );
-      }
+    let submitButton = (
+      <Button
+        variant="contained"
+        color="primary"
+        onClick={currentlyEditing ? handleUpdate : handleSubmit}
+        disabled={submitButtonDisabled}
+      >
+        {currentlyEditing ? 'Update' : 'Submit'}
+      </Button>
+    );
+
+    let newsJSX;
+
+    // If the user is online
+    if (isOnline || (!isOnline && props.authentication)) {
+      newsJSX = (
+        <>
+          {/* Button to Create Posting */}
+          <Fab variant="extended" color="primary" onClick={handlePostClick} style={styles.fab}>
+            <PostAddIcon />
+            Post Listing
+          </Fab>
+
+          <Grid container justify="center">
+            {/* Search */}
+            <Grid item xs={12} md={12} lg={8}>
+              <Grid
+                container
+                alignItems="baseline"
+                justify="center"
+                style={styles.searchBar}
+                spacing={5}
+              >
+                <Grid item xs={10} sm={8} md={8} lg={6}>
+                  <TextField
+                    id="search"
+                    label="Search news"
+                    value={search}
+                    onChange={(event) => {
+                      setSearch(event.target.value);
+                    }}
+                    margin="normal"
+                    fullWidth
+                  />
+                </Grid>
+              </Grid>
+            </Grid>
 
-      let submitButton = (
-        <Button
-          variant="contained"
-          color="primary"
-          onClick={this.handleSubmit.bind(this)}
-          disabled={submitButtonDisabled}
-        >
-          Submit
-        </Button>
-      );
-      let editButton = (
-        <Button
-          variant="contained"
-          color="primary"
-          onClick={this.handleUpdate.bind(this)}
-          disabled={submitButtonDisabled}
-        >
-          Update
-        </Button>
-      );
+            {/* NOTE: leaving helper text for now in case
+            that is better than disabling submit button */}
+            {/* Create Posting */}
+            <Dialog open={openPostActivity} fullWidth>
+              <DialogTitle> Post on Student News </DialogTitle>
+              <DialogContent>
+                <Grid container>
+                  {/* CATEGORY ENTRY */}
+                  <Grid item>
+                    <TextField
```

Comment:
You should go through all the TextFields and make sure they have the `variant="filled"` prop.